src/proxyless/utils/cdp.ts

```diff
@@ -24,6 +24,10 @@ export function isRequest (event: RequestPausedEvent): boolean {
     return event.responseStatusCode === void 0;
 }
 
+export function isRedirect (event: RequestPausedEvent): boolean {
```

Comment:
You can drop this function and use `isRedirectStatusCode` from the `testcafe-hammerhead` module.
https://github.com/DevExpress/testcafe/pull/7366/files#diff-fe9ea2288fb9df9ede95df2c68c40a24c03563af266672f0e2ecfd135662a106R22